Formula/bochs.rb

```diff
@@ -28,12 +29,14 @@ class Bochs < Formula
 
   def install
     args = %W[
+      CXXFLAGS=-std=c++11
```

Comment:
Is this needed? Seemed to work fine when I tested earlier. If so, maybe this would be cleaner:

```suggestion
    ENV.cxx11
    args = %W[
```

Reply:
Yeah, seems to work without C++11 for me as well, and I can't remember why I included it initially.